### PR TITLE
[#100] Update testcontainers version to 0.12.0

### LINKED ISSUE #100: check dependabot alerts
image

https://github.com/nspcc-dev/neofs-http-gw/security/dependabot/go.sum/github.com%2Fopencontainers%2Frunc/open

### PR DESCRIPTION
**Pull request by KirillovDenis**
closes #100 

Signed-off-by: Denis Kirillov <denis@nspcc.ru>

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -661,16 +670,19 @@ github.com/opencontainers/runc v0.0.0-20190115041553-12f6a991201f/go.mod h1:qT5X
 github.com/opencontainers/runc v0.1.1/go.mod h1:qT5XzbpPznkRYVz/mWwUaVBUv2rmF59PVA73FjuZG0U=
 github.com/opencontainers/runc v1.0.0-rc8.0.20190926000215-3e425f80a8c9/go.mod h1:qT5XzbpPznkRYVz/mWwUaVBUv2rmF59PVA73FjuZG0U=
 github.com/opencontainers/runc v1.0.0-rc9/go.mod h1:qT5XzbpPznkRYVz/mWwUaVBUv2rmF59PVA73FjuZG0U=
-github.com/opencontainers/runc v1.0.0-rc93 h1:x2UMpOOVf3kQ8arv/EsDGwim8PTNqzL1/EYDr/+scOM=
 github.com/opencontainers/runc v1.0.0-rc93/go.mod h1:3NOsor4w32B2tC0Zbl8Knk4Wg84SM2ImC1fxBuqJ/H0=
```
Comment (532910) [inline go.sum:673]: Not sure if I understand, but `v1.0.0-rc93` shouldn't be user, should it?

Comment (532910) [reply go.sum:673]: Possibly, the same as `v0.1.1`?

**Review by 532910 — COMMENTED** [review]
Should opencontainers/image-spec be also updated?

File: go.sum (hunk 1 above)
Comment (KirillovDenis) [reply go.sum:673]: It seems [testcontainers-go](https://github.com/testcontainers/testcontainers-go) maintainers should also update `github.com/Microsoft/hcsshim@v0.8.16` package.  The `v1.0.0-rc93` is transitive dependency: `github.com/testcontainers/testcontainers-go@v0.12.0 -> github.com/Microsoft/hcsshim@v0.8.16 -> github.com/opencontainers/runc@v1.0.0-rc93`.

Comment (alexvanin) [reply go.sum:673]: Let's merge and see if dependabot triggers on hash of `go.mod` or not.

> It seems testcontainers-go maintainers should also update github.com/Microsoft/hcsshim@v0.8.16 package. The v1.0.0-rc93 is transitive dependency: github.com/testcontainers/testcontainers-go@v0.12.0 -> github.com/Microsoft/hcsshim@v0.8.16 -> github.com/opencontainers/runc@v1.0.0-rc93.

Not really. If I understand correctly, `go mod` stores hashes of `go.mod` files for some sort of dependency consistency. However application _will not_ use old version of libraries if there is a newer one (within a major version). So you can see only the hash of the latest `runc` library source in go.sum and no such hashes for older versions:
```
github.com/opencontainers/runc v1.0.2 h1:opHZMaswlyxz1OuGpBE53Dwe4/xF7EZTY0A2L/FpCOg=
```

**Review by alexvanin — APPROVED** [review]

**Pull request merged**